### DISCUSSION TITLE
Which MINING POOL we can connect through btcpayserver ?

### DISCUSSION THREAD

**Discussion by hrshit-Glix12** [btcpayserver/btcpayserver#3939] [Plugins & integrations]
It would be good if you can share, which mining pool we can connect through btcpayserver.
Few other questions as below:

btcpayserver can able to connect with any mining pool or not ? if yes, then how we can able to connect.

**Comment by pavlenex** [comment]
Why would you want to connect BTCPay with a mining pool? That's not what BTCPay is for, you may want to do a bit of reading on our docs.btcpayserver.org

**Comment by bolatovumar** [comment]
You wouldn't use BTCPay Server to connect to a mining pool, that's not what it's for. Of course, you could use BTCPay to set up a wallet to accept mining pool payouts into but any other wallet would work just as well for this purpose, BTCPay is a bit of an overkill for this use case.
